src/test/java/com/artipie/rpm/RpmITCase.java

```diff
@@ -0,0 +1,120 @@
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm;
+
+import com.amihaiemil.eoyaml.Yaml;
+import com.artipie.ArtipieServer;
+import com.artipie.asto.Key;
+import com.artipie.asto.fs.FileStorage;
+import com.artipie.asto.test.TestResource;
+import com.artipie.http.rs.RsStatus;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.nio.file.Path;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+/**
+ * IT case for RPM repository.
+ * @since 0.12
+ * @checkstyle MagicNumberCheck (500 lines)
+ */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
+public final class RpmITCase {
+
+    /**
+     * Temporary directory for all tests.
+     * @checkstyle VisibilityModifierCheck (3 lines)
+     */
+    @TempDir
+    Path tmp;
+
+    /**
+     * Tested Artipie server.
+     */
+    private ArtipieServer server;
+
+    /**
+     * Artipie server port.
+     */
+    private int port;
+
+    @BeforeEach
+    void init() throws IOException {
+        this.server = new ArtipieServer(this.tmp, "my-rpm", this.configs());
+        this.port = this.server.start();
+    }
+
+    @Test
+    void addsRpm() throws Exception {
+        final HttpURLConnection con = (HttpURLConnection) new URL(
+            String.format("http://localhost:%s/my-rpm/time-1.7-45.el7.x86_64.rpm", this.port)
+        ).openConnection();
+        con.setRequestMethod("PUT");
+        con.setDoOutput(true);
+        try (OutputStream out = con.getOutputStream()) {
+            final byte[] input = new TestResource("rpm/time-1.7-45.el7.x86_64.rpm").asBytes();
+            out.write(input, 0, input.length);
+        }
+        MatcherAssert.assertThat(
+            "Response status is 202",
+            con.getResponseCode(),
+            new IsEqual<>(Integer.parseInt(RsStatus.ACCEPTED.code()))
+        );
+        MatcherAssert.assertThat(
+            "Repository xml indexes are created",
+            new FileStorage(this.tmp).list(new Key.From("my-rpm/repodata")).join().size(),
+            new IsEqual<>(4)
+        );
+        con.disconnect();
+    }
+
+    @AfterEach
+    void close() {
+        this.server.stop();
+    }
+
+    private String configs() {
```

Comment:
@olenagerasimova guess there is typo and this method's name is supposed to be `config`, not `configs`, isn't it?